crates/sui-storage/src/lock_service.rs

```diff
@@ -488,74 +488,86 @@ impl LockService {
         refs: Vec<ObjectRef>,
         tx_digest: TransactionDigest,
     ) -> SuiResult {
-        let (os_sender, os_receiver) = oneshot::channel::<SuiResult>();
-        // NOTE: below is blocking, switch to Tokio channels which are async?
-        self.inner
-            .sender()
-            .send(LockServiceCommands::Acquire {
-                refs,
-                tx_digest,
-                resp: os_sender,
-            })
-            .await
-            .expect("Could not send message to inner LockService");
-        os_receiver
-            .await
-            .expect("Response from lockservice was cancelled, should not happen!")
+        exec_client_future(async move {
+            let (os_sender, os_receiver) = oneshot::channel::<SuiResult>();
+            // NOTE: below is blocking, switch to Tokio channels which are async?
```

Comment:
The comment seem outdated

Reply:
agreed - will remove as long as i'm polluting the blame anyway.